www/index.php

```diff
@@ -299,24 +304,6 @@ function renderTemplate($file, $params = [])
 $page->addStyleDeclaration(file_get_contents(__DIR__ . '/css/go.css'));
 $page->addHeadLink($lilurl->getBaseUrl(), 'home');
 
-$page->addScriptDeclaration("
-require(['jquery'], function($) {
-    $(function() {
-        $('#moreOptions').hide();
```

Comment:
Only remove the `$('#showMoreOptions').click(function() {...}` block.  The $out code is not related the show more options   functionality and is still needed.